src/System.Private.CoreLib/shared/System/Reflection/SignatureConstructedGenericType.cs

```diff
@@ -12,9 +12,21 @@ internal sealed class SignatureConstructedGenericType : SignatureType
     {
         internal SignatureConstructedGenericType(Type genericTypeDefinition, Type[] genericTypeArguments)
```

Comment:
Nit: Rename this to `typeArguments` so that it has the same name as the public factory method? (The name shows up in the exception message.)

Reply:
Ok, renamed.